charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -39,10 +46,16 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
+        {{- if .Values.logForward.create }}
+        {{- include "fluent-bit-sidecar-pod-uid" . | indent 8 }}
+        {{- end }}
```

Comment:
This can be passed through values.yaml as additional env vars

Reply:
fluent-bit-sidecar-pod-uid removed and UID passing through values